PrebidMobile/API1.0Demo/src/main/java/org/prebid/mobile/app/DemoActivity.java

```diff
@@ -66,7 +70,113 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             createMoPubBanner(intent.getStringExtra(Constants.AD_SIZE_NAME));
         } else if ("MoPub".equals(intent.getStringExtra(Constants.AD_SERVER_NAME)) && "Interstitial".equals(intent.getStringExtra(Constants.AD_TYPE_NAME))) {
             createMoPubInterstitial();
+        } else if ("DFP".equals(intent.getStringExtra(Constants.AD_SERVER_NAME)) && "Native".equals(intent.getStringExtra(Constants.AD_TYPE_NAME))) {
```

Comment:
I still can not get successful result neither from DFP nor MoPub. 
PBS - `<head><title>400 Bad Request</title></head>`

Reply:
Some configuration was not right and something's not working with PBS, trying to find out why now.